go.mod

```diff
@@ -2,16 +2,27 @@ module github.com/siderolabs/talemu
 
 go 1.22.5
 
-// forked go-yaml that introduces RawYAML interface, which can be used to populate YAML fields using bytes
-// which are then encoded as a valid YAML blocks with proper indentation
-replace gopkg.in/yaml.v3 => github.com/unix4ever/yaml v0.0.0-20220527175918-f17b0f05cf2c
+replace (
+	// forked go-yaml that introduces RawYAML interface, which can be used to populate YAML fields using bytes
+	// which are then encoded as a valid YAML blocks with proper indentation
+	gopkg.in/yaml.v3 => github.com/unix4ever/yaml v0.0.0-20220527175918-f17b0f05cf2c
```

Comment:
can't we use the project that implements a basic apiserver, rather than importing all k8s? i can't recollect the project name

Reply:
it was `sample-apiserver`, I've tried that one. It's missing lots of things required for the core API server which are not documented anywhere.
So I thought that it's easier to just run a real API server, which sets up everything properly.

Reply:
not that one, we can reconsider once I find the project again :sweat_smile: